src/frontend/js/utils/api/web-analytics/base.ts

```diff
@@ -0,0 +1,19 @@
+import { WebAnalyticsAPI } from 'types/WebAnalytics';
+
+/**
+ * Base Web Analytics Richie Web Analytics API Implementation
+ * Implementation used when not providing a specific web analytics provider
+ */
+const BaseWebAnalyticsApiInterface = (): WebAnalyticsAPI => {
+  return {
+    type: (): string => {
+      return 'base';
+    },
+    // eslint-disable-next-line no-unused-vars
+    send_enrolled_event: async (): Promise<any> => {
+      // 'No web analytics provider found, no enrolled event will be sent.'
+    },
+  };
+};
+
+export default BaseWebAnalyticsApiInterface;
```

Comment:
In the case of web analytics, It does make sense to declare a `base` interface until we need to fake it for development purpose.

Reply:
Removed.
Added a test when there isn't any provider. `unknown_provider.spec.ts`